src/graphql.ts

```diff
@@ -172,13 +172,13 @@ directive @computed on FIELD_DEFINITION
 
       Object.defineProperties(node, properties);
 
-      let computed = type.computed.reduce((props, compute) => {
+      let computed = type.computed.filter(compute => !vertex.data[compute.name]).reduce((props, compute) => {
```

Comment:
It just occurred to me, should we use `computer.name in vertex.data` for the filter condition in case the data has an explicit `null` or `undefined`, `0` or `""`?

Reply:
@cowboyd I think you have a point, that needs fixed.  Also what would happen if, after the filter, an empty object was left.

Reply:
what would we use instead of compute.name?

The `preset` :).

Reply:
Sorry, I meant the filter condition:

```ts
let computed = type.computed.filter(compute => !(compute.name in vertex.data)).reduce((props, compute) => {
```

Reply:
Or maybe even `!Object.keys(vertex.data).includes(compute.name)`